mitiq/shadows/executor_functions.py

```diff
@@ -0,0 +1,47 @@
+from mitiq.typing import QuantumResult, MeasurementResult
+from mitiq import Executor
+import cirq 
+from typing import List, Tuple, Dict, Any, Optional, Union
+import qiskit
+from qiskit_aer import Aer
+from mitiq.interface.mitiq_qiskit.conversions import to_qiskit
+import numpy as np
+from tqdm.auto import tqdm, trange
+
+
+# cirq simulator
+def cirq_simulator_shadow_executor_fn(circuits: List[cirq.Circuit], max_batch_size:int=10000) -> List[MeasurementResult]:
```

Comment:
Missing docstrings. Not sure this function should live in the codebase though.

Reply:
yes, it would work